interview-questions.md

```diff
@@ -15,54 +15,54 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is the version control of code. Github is the host (GUI)and Git is the different versions of source code
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: logical and comparison
 
-  Researched answer:
+  Researched answer: In JavaScript, logical operators are used for boolean logic where a boolean value may be returned depending on the outcome of an expression.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: index is the numerical place of a data type, index represents numerical place while value is the input/element
 
-  Researched answer:
+  Researched answer: A index is a read-only property. It contains the position of a regular expression match within a string while a value is used to define what the actual variable type value is being held in the elements/indexes.
 
 
 
 4. What is iteration?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: like a sequnce of functions
 
+  Researched answer: An object which defines a sequence and potentially a return value upon its termination.
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: Pair programming typically consists of 2 or more people with one operating as a driver(person that inputs the code) and the other as the navigator(person that dircts and tell the driver what codes to put in)
 
-  Researched answer:
+  Researched answer: Pair programming is an Agile software development technique originating from Extreme programming (XP) in which two developers team together on one computer. The two people work together to design, code and test user stories.
 
 
 
 ## Looking Ahead: Terms for Next Week
 
 Research and define the following terms to the best of your ability.
 
-1. Higher Order Functions:
+1. Higher Order Functions: A “higher-order function” is a function that accepts functions as parameters and/or returns a function.
 
-2. Jest:
+2. Jest: is a JavaScript test runner, that is, a JavaScript library for creating, running, and structuring tests
 
-3. Objects:
+3. Objects: are containers for named values called properties.
 
-4. Method:
+4. Method: JavaScript methods are actions that can be performed on objects.
+Is a property containing a function definition.
 
-5. Classes:
+5. Classes: Classes are a template for creating objects. They encapsulate data with code to work on that data.
```

Comment:
Venessa - for the researched answer, please rewrite each answer in your own words.  We want to make sure you understand the concepts.  Also, for future assessments, do your best to expand your own answers to more than a sentence or two. A couple of words or one sentence is the bare minimum required. You want it to be conversational and show you are interested in the job.